tests/utils/utils.go

```diff
@@ -6,15 +6,37 @@ package tests
 import (
 	"bufio"
 	"context"
+	"crypto/ecdsa"
+	"encoding/hex"
+	"fmt"
+	"math/big"
 	"os"
 	"os/exec"
 	"strings"
 
+	"github.com/ava-labs/avalanchego/utils/logging"
+	"github.com/ava-labs/awm-relayer/config"
+	"github.com/ava-labs/awm-relayer/peers"
+	"github.com/ava-labs/teleporter/tests/interfaces"
+	"github.com/ava-labs/teleporter/tests/utils"
+	teleporterTestUtils "github.com/ava-labs/teleporter/tests/utils"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/crypto"
 	"github.com/ethereum/go-ethereum/log"
 	. "github.com/onsi/gomega"
 )
 
-func RunRelayerExecutable(ctx context.Context, relayerConfigPath string) (*exec.Cmd, context.CancelFunc) {
+var (
+	storageLocation = fmt.Sprintf("%s/.awm-relayer-storage", os.TempDir())
```

Comment:
Could you add a comment stating that it's okay to use /tmp for tests. I saw this at first and was confused why we were using it for storage.